### DISCUSSION TITLE
Integration with js-joda

### DISCUSSION THREAD

**Discussion by Newbie012** [prisma/prisma#4797] [Q&A] [answered]
I am using [js-joda](https://js-joda.github.io/js-joda/) as my library for processing date stuff.

While the other date libraries are acting as wrappers of the native javascript date object, js-joda has its own implementation of doing so.

The problem occurs when Prisma assumes that since created_at is `DateTime` based on `prisma.schema`, then it should be Date. Therefore, I when I want to create, update or filter, I won't be able to use js-joda.

Is there a way I can tell Prisma to parse the Date as Instant/LocalDate/LocalDateTime and serialize it back to date when its needed?

**Comment by nikolasburk** [comment, marked as answer]
Hey @Newbie012, thanks a lot for raising this! Using `js-joda` indeed is not natively supported by Prisma at the moment – if you'd like to see an integration that makes handling dates easier I suggest that you [open a feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this, that way our product and engineering team can look into it and incorporate it in our [roadmap](https://pris.ly/roadmap)!

As a workaround I guess you'd need to convert the `Date` objects used by Prisma to the ones that are compatible with `js-joda`.

Please let me know if that helps or if you have any further questions, always happy to help 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
